### PR TITLE
Disabling of stuff

### PR DESCRIPTION
**Pull request by PatrykSzwer**
For #116 

It's not done yet. For now on It's only front-end/configuration part. Now We need to implement handling for disabling in WebsiteProvider. As We discussed with @un-tone, He'll code that part.

### CODE REVIEW

File: src/WebsiteEditor/ViewModels/BlendingPointsPage.json [hunk 1]
```diff
@@ -19,7 +20,8 @@
           "UrlKey$": "",
           "ForeignUrl$": "",
           "SortNumber$": 0,
-          "Delete$": 0
+          "Delete$": 0,
```
Comment (un-tone) [inline src/WebsiteEditor/ViewModels/BlendingPointsPage.json]: JSON properties that are bound to actions in code-begind should have a Trigger$ suffix (https://starcounter.gitbooks.io/guidelines/content/coding_style/json-common-practices.html)

Comment (un-tone) [inline src/WebsiteEditor/ViewModels/BlendingPointsPage.json]: I will fix it myself. It is just to let you know

Comment (PatrykSzwer) [inline src/WebsiteEditor/ViewModels/BlendingPointsPage.json]: Thanks, I'll keep this in mind.

**Comment by un-tone** [comment]
@PatrykSzwer I finished with disabling stuff implementation. Please review and check. I'm currently working on tests covering this feature.

File: src/WebsiteProvider/Api/PinningHandlers.cs [hunk 2]
```diff
@@ -206,44 +221,224 @@ public void UnmapSurface(ulong templateId)
...
+                    foreach (var otherTokens in customCatchingRuleMappings.Select(x => x.Token).Distinct())
+                    {
+                        if (!Blender.IsMapped(mapInfo.ForeignUri, otherTokens))
+                        {
+                            Blender.MapUri(mapInfo.ForeignUri, otherTokens, false, true);
+                        }
+                    }
+                }
+
+                // now do map current Pinning Rule's URI to the current token and save mapping info.
```
Comment (PatrykSzwer) [inline src/WebsiteProvider/Api/PinningHandlers.cs]: If I can just add a suggestion, I would rewrite this sentece, It's not really readable (unless for me).

Comment (un-tone) [inline src/WebsiteProvider/Api/PinningHandlers.cs]: Please check again

**Comment by un-tone** [comment]
All required tests are added.

**Pull request merged**